src/shared/components/ui/BlockSection/BlockSection.js

```diff
@@ -36,16 +34,17 @@ function BlockSection({
   url,
   reverse,
   list
-}) {
-  const elementClasses = cssClass({
-    'block-section__panel--reverse': reverse === true
+}) => {
+  const elementClasses = useStyles({
+    [styles['block-section__panel']]: true,
+    [styles['block-section__panel--reverse']]: reverse === true
```

Comment:
w sumie mogłoby być bez porównania :p

Reply:
racja, zostawiłem po prostu jak było i nawet na to uwagi nie zwróciłem